notebooks/official/workbench/inventory-prediction/inventory_prediction.ipynb

```diff
@@ -447,7 +400,7 @@
         "\n",
         "The following cell integrates with BigQuery data from the same project through the Vertex AI's \"BigQuery in Notebooks\" integration. It can run an SQL query as it would run in the BigQuery console. \n",
         "\n",
-        "*Note:* This feature only works in a notebook running on a Vertex AI Workbench managed-notebook instance."
+        "**Note:** This feature only works in a notebook running on a Vertex AI Workbench managed-notebook instance."
```

Comment:
RE: "Vertex AI Workbench managed-notebook instance", is that referencing "Vertex AI Workbench instances"?? There is no reference to "Vertex AI Workbench managed-notebook instance" in the documentation.

Reply:
I think they are referencing "Vertex AI managed notebook instances " which will be deprecated.
 https://cloud.google.com/vertex-ai/docs/workbench/notebook-solution#managed